src/components/PropertyMap.tsx

```diff
@@ -46,7 +46,7 @@ import '@mapbox/mapbox-gl-geocoder/dist/mapbox-gl-geocoder.css';
 import { MapLegendControl } from './MapLegendControl';
 import { createPortal } from 'react-dom';
 import { Tooltip } from '@nextui-org/react';
-import { Info, X } from '@phosphor-icons/react';
+import { Info, MapPinArea, X } from '@phosphor-icons/react';
```

Comment:
How do we feel about using this Icon?

Reply:
@DonovanAndrews300 I tested out your PR in the Vercel preview env. The button does work; it toggles the map view. However, the ticket requirement asks that we use MapTiler's style switcher control.

Here's an example from https://docs.maptiler.com/sdk-js/examples/control-style-switcher/

![Screen Shot 2024-12-15 at 3 49 31 PM](https://github.com/user-attachments/assets/be61daa5-883a-4367-bc4c-26093a86a51a)
![Screen Shot 2024-12-15 at 3 49 43 PM](https://github.com/user-attachments/assets/e32e87d4-d917-4490-9854-6bbab22d5df1)

Reply:
Thanks Gary I got the change coming in with the switch controller shortly.